pkg/api/v1alpha1/cluster_types.go

```diff
@@ -683,6 +684,12 @@ const (
 	Kube126 KubernetesVersion = "1.26"
 )
 
+// KubeVersionToValidSemver converts kube version to semver for comparisons.
```

Comment:
move it here to avoid import cycle

Reply:
Where was this moved from? I can't see anything that would force it to need to be in this package.

Reply:
https://github.com/aws/eks-anywhere/blob/8acc8aba5bd9f429242abf463f6bc33a2e674dba/pkg/semver/utils.go#L7
This is to avoid import cycle between semver and pkg/api/v1alpha1. `KubeVersionToValidSemver` is the only place that semver import v1alpha1